### DISCUSSION TITLE
[Mac ARM] SOS with my disasterous directory cess & dotfiles mess (open to paid help)!

### DISCUSSION THREAD

**Discussion by BuildBackBuehler** [marlonrichert/zsh-snap#237] [General]
Been at this a year, I wanted to keep a strict straight-line PATH but because NVM and Homebrew don't play nice – and my brew & git seem to fight/cause errors, my directories are absolute chaos and its been a futile battle to fix the jankiness. 

Unfortunately zsh-autocomplete's infinite loop issue had me thinking it must be my setup so it was no longer "if it ain't broke, don't fix it" -> nuked it -> nothing was the same ever since!! Wooh.

My original diligent research into setup org. didn't pay off. For example, I suppose because Homebrew + /usr/local/ don't play nice due to permission issues, I tried to make Brew's base in the suggested /opt/homebrew locality...but now there's /opt/homebrew/ and a mess within it of /bin, /share, /opt (yes again with an extra /share!) and so on. Git & Homebrew seem to duke it out, rarely errors but redundancies on redundancies. Atop of that I have everything in $HOME as far as git & zsh config files, npm/pnpm, miniforge and sooo many misplaced plugins or mini projects. An absolute monstrosity.

_____________

TL;DR 

- Looking for hands-on help cleaning up my coding setup **(open to paying!!!)**. Not confident in my ability to choose someone's DOT files for a blueprint. 
- Blueprint would optimally have foundation in Homebrew stylized w/ powerlevel10k. I use zsh, docker (SSH into Synology NAS), nvm/npm/pnpm (website "coding") primarily. Most important deetz...

    - I'd like a barebones-sans-ohmyzsh zsh that relies on znap + autocomplete/suggestions (history)/syntax highlighting. 
    - Lazy loaded autocomplete functions when I'm in the other environments (docker, npm, etc.)
- My dreamiest setup could also autoswap to a Kubernetes setup with minikube/ps-1/kubectx/kubens/stern etc., attempted this w/ a keybind/powerlevel9k prompt triggered w/ a kubectl-type command. Was complex and injury-prone.

It sounds so simple yet I've spent days and alas, broken and ready to rip my hair out. 

Cheers for whatever support (emotional or otherwise 🥴) provided!

**Comment by marlonrichert** [comment]
Try my [Zsh Launchpad](https://github.com/marlonrichert/zsh-launchpad).
